lib/templates/lab3/notifier.dart

```diff
@@ -20,15 +20,21 @@ class WaterIntakeNotifier extends StateNotifier<double> {
 
   void _loadWaterIntake() async {
     // TODO - Load the water intake from _localStorageService using await
-    // state =
+    state = await _localStorageService.getWaterIntake();
   }
 
   void increment(double amount) async {
     state += amount;
     // TODO - Save the water intake into _localStorageService using saveWaterIntake
+    if (state == 3000) {
```

Comment:
Why not >=

Reply:
It's a mistake that I noticed only after the submission, if statement was supposed to be in the beginning of func, just so value of drank water wouldn't exceed 3000 ml

Reply:
It is okey, I just want to highlight